src/guide/kuzzle-backend-setup/aws-marketplace.md

```diff
@@ -4,4 +4,63 @@ algolia: true
 title: AWS Marketplace
 description: AWS Marketplace
 order: 1200
----
\ No newline at end of file
+---
+
+# AWS Marketplace
+<div class="alert alert-info">
+To follow this tutorial, you need a __valid AWS account__.
+</div>
+
+_In this guide, you'll learn where and how to use our AWS Marketplace AMI. It's a good way to test Kuzzle in a cloud environment. In addtion, we recommend you to use our [Kuzzle Admin Console](http://console.kuzzle.io), the most comfy way to play with Kuzzle._
+
+## Get the AMI
+
+Our AMI is stored on AWS Marketplace. It's setup with:
+  * Ubuntu (__16.04__)
+  * Kuzzle (__latest__) with MQTT protocol support.
+  * Elasticsearch (__v5.4.1__).
+  * Redis (__v3.2.12__).
+
+Go to the marketplace and type __kuzzle__ in the search form.
+Choose your Amazon EC2 instance type (the minimal requirement is a __t2-medium__).
+Recover the public IP or the hostname provided by AWS before continue.
+Check if Kuzzle is up and running by simply performing the following HTTP request::
+
+```sh
+$ curl http://yourInstanceIpOrHostname:7512/_serverInfo\?pretty
+{
+  "requestId": "9b07a095-7143-49a5-9079-a34e937fdf3e",
+  "status": 200,
+  "error": null,
+  "controller": "server",
+  "action": "info",
+  "collection": null,
+  "index": null,
+  "volatile": null,
+  "result": {
+    # Some stuff
+  }
+}
+```
+
+You should see some informations about your Kuzzle Server.
+If not, wait a few minutes and retry the request.
+
+## Connect with default credentials
+
+Open the [Kuzzle Admin Console](http://console.kuzzle.io) and fill the form with the address of your Kuzzle instance. There is a default admin user with __ec2-user__ as username.
+Associated password is your unique instance ID. You can get it from the EC2 AWS Console, it looks like this __i-xxxxxxxxxxxxxxxxx__. Don't worry, you can modify it or create an other
```

Comment:
`create an other admin user` => `create another admin user`

But I don't know how creating a new user will make the existing one more secure. I'd prefer that you explain how to easily change that existing user's password.

Reply:
It's not about security. The default password is only known by the user (instance ID).

Reply:
Ok then the `Don't worry` part has to go.

Reply:
done :+1: